package exception;

public class FoundUserException extends RuntimeException{

    public FoundUserException(String message) {
        super(message);
    }
}
